autometa/common/kmers.py

```diff
@@ -127,7 +129,7 @@ def load(kmers_fpath):
         raise FileNotFoundError(kmers_fpath)
     try:
         df = pd.read_csv(kmers_fpath, sep="\t", index_col="contig")
-    except ValueError:
+    except (ValueError, TypeError):
```

Comment:
Is this another error that occurs from `pd.read_csv(...)`?

Reply:
Yeah, I encountered this when I was passing in a Mock or MagicMock object to it. It seems it is raised when the input file is not a correct object

Reply:
It sounds like this should be handled separately, since the error is a result from a different input. I.e. this error is not a result of the `contig` column being absent.